pkg/ebpftracer/c/headers/types.h

```diff
@@ -5,6 +5,7 @@
 #include <vmlinux_missing.h>
 
 #include <common/consts.h>
+#include <common/rate_limit.h>
```

Comment:
not implemented yet right?

Reply:
Left over from other POC. In this PR I tried rate limiter based approach https://github.com/castai/kvisor/pull/554
But having maps and scrape periodically is still better